pkg/engine/operational_rule/operational_rule.go

```diff
@@ -132,7 +132,7 @@ func (ctx OperationalRuleContext) CleanProperty(step knowledgebase.OperationalSt
 	if err != nil {
 		return err
 	}
-	prop := path.Get()
+	prop, _ := path.Get()
```

Comment:
I'm not sure where you get that from?

The old behavior was to simply return `nil` from `PropertyPath.Get()` if either the path wasn't found or the value was `nil`. The new interface adds the boolean in the return to clarify the case where a path was found, but its content is `nil`. The pattern of ignoring the boolean was introduced to avoid accidentally changing behavior in areas of the engine in which `PropertyPath` is already being used.

Reply:
For example, with the way that you changed 
```diff
func (i *arrayIndexPathItem) Get() (any, bool) {
+	if !i.a.IsValid() || !reflectutil.IsAnyOf(reflectutil.GetConcreteElement(i.a), reflect.Slice, reflect.Array) {
+		return nil, false
+	}
+	if i.a.Len() <= i.index {
+		return nil, false
+	}
	return i.a.Index(i.index).Interface()
```
This changes what would have previously `panic`d into one that returns false.